### DISCUSSION TITLE
What is the "DeviceTenant" in the CumulocityConnectorConfig.json file?

### DISCUSSION THREAD

**Discussion by DriesDebouver** [hms-networks/flexy-cumulocity-connector#21] [Q&A]
What is the "DeviceTenant" from the Ewon Flexy?
Is it the name when registering a new device in Cumulocity?
Is it the Ewon serial number?
Is it the Ewon name itself?

![image](https://user-images.githubusercontent.com/28297720/188451198-7281a3aa-eaca-4cbe-9593-8c861ddaa9a8.png)

**Comment by StefanRuhmkorf** [comment]
The DeviceTenant is the ID of the Cumulocity tenant. 
This ID will be delivered back from Cumulocity during the bootstrap process for authentication.
